runtime/src/lib.rs

```diff
@@ -93,7 +93,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("vitalam-node"),
     impl_name: create_runtime_str!("vitalam-node"),
     authoring_version: 1,
-    spec_version: 270,
```

Comment:
we haven't deployed this yet so I think on this occasion we can stick to `270`. Need to work out a runtime versioning system

Reply:
rolled back to `270`